core/email_api.php

```diff
@@ -1365,6 +1369,13 @@ function email_send( EmailData $p_email_data ) {
 	$t_mail->AddCustomHeader( 'Auto-Submitted:auto-generated' );
 	$t_mail->AddCustomHeader( 'X-Auto-Response-Suppress: All' );
 
+	if( isset($t_email_data->metadata['cc']) && $t_email_data->metadata['cc'] )
+		foreach( $t_email_data->metadata['cc'] as $cc )
+			$t_mail->addCC(trim($cc));
+	if( isset($t_email_data->metadata['bcc']) && $t_email_data->metadata['bcc'] )
+		foreach( $t_email_data->metadata['bcc'] as $bcc )
+			$t_mail->addBCC(trim($bcc));
+
```

Comment:
Coding guidelines: always use braces `{}` after if and foreach, even when there is a single statement.
Also make sure you have a space after `(` and before `)`